recipes-security/refpolicy/refpolicy-mcs-2.20141203/policy/modules/services/xenpmd.fc

```diff
@@ -18,5 +18,7 @@
 #
 #############################################################################
 
-/usr/sbin/xcpmd		--	gen_context(system_u:object_r:xenpmd_exec_t,s0)
-/var/run/xcpmd\.pid    --      gen_context(system_u:object_r:xenpmd_var_run_t,s0)
+/usr/sbin/xcpmd                -- gen_context(system_u:object_r:xenpmd_exec_t,s0)
+/var/run/xcpmd\.pid            -- gen_context(system_u:object_r:xenpmd_var_run_t,s0)
+/usr/share/xcpmd               -d gen_context(system_u:object_r:xenpmd_etc_t,s0)
+/usr/share/xcpmd/default.rules -- gen_context(system_u:object_r:xenpmd_etc_t,s0)
```

Comment:
Not to be a pain and I really want to get this one in but does default.rules need the period escaped? It seems to be everywhere else in those policies.